src/components/interfaces/MOBILE_API.xml

```diff
@@ -5369,65 +5454,149 @@
+      Legacy / v1 Protocol implementation; responds to EncodedSyncPData.
+      *** DEPRECATED ***
     </description>
     <param name="data" type="String" maxlength="1000000" minsize="1" maxsize="100" mandatory="true" array="true">
       <description>Contains base64 encoded string of SyncP packets.</description>
     </param>
     <param name="URL" type="String" maxlength="1000" mandatory="false">
       <description>
-      	If blank, the SyncP data shall be forwarded to the app.
-      	If not blank, the SyncP data shall be forwarded to the provided URL.
+        If blank, the SyncP data shall be forwarded to the app.
+        If not blank, the SyncP data shall be forwarded to the provided URL.
       </description>
     </param>
     <param name="Timeout" type="Integer" minvalue="0" maxvalue="2000000000" mandatory="false">
       <description>
-      	If blank, the SyncP data shall be forwarded to the app.
-      	If not blank, the SyncP data shall be forwarded with the provided timeout in seconds.
+        If blank, the SyncP data shall be forwarded to the app.
+        If not blank, the SyncP data shall be forwarded with the provided timeout in seconds.
       </description>
     </param>
   </function>
 
-<function name="SetAudioStreamingIndicator" functionID="SetAudioStreamingIndicatorID" messagetype="request">
-   <description>Sets the icon properly to the current audio playback of the app.</description>
-   <param name="audioStreamingIndicator" type="AudioStreamingIndicator" mandatory="true">
-    <description>
-     See AudioStreamingIndicator.
-    </description>
-   </param>
-</function>
+  <function name="SubscribeWayPoints" functionID="SubscribeWayPointsID" messagetype="request">
+        <description>To subscribe in getting changes for Waypoints/destinations</description>
+    </function>
 
-<function name="SetAudioStreamingIndicator" functionID="SetAudioStreamingIndicatorID" messagetype="response">
-   <param name="success" type="Boolean" platform="documentation">
-    <description>true if successful; false, if failed </description>
-   </param>
-   <param name="resultCode" type="Result" platform="documentation">
-     <description>See Result</description>
-     <element name="SUCCESS"/>
-     <element name="INVALID_DATA"/>
-     <element name="OUT_OF_MEMORY"/>
-     <element name="TOO_MANY_PENDING_REQUESTS"/>
-     <element name="APPLICATION_NOT_REGISTERED"/>
-     <element name="GENERIC_ERROR"/>
-     <element name="REJECTED"/>
-     <element name="IGNORED"/>
-     <element name="DISALLOWED"/>
-     <element name="USER_DISALLOWED"/>
-   </param>
-   <param name="info" type="String" maxlength="1000" mandatory="false" platform="documentation">
-     <description>Provides additional human readable info regarding the result.</description>
-   </param>
-</function>
+  <function name="SubscribeWayPoints" functionID="SubscribeWayPointsID" messagetype="response">

```

Comment:
@LuxoftAKutsan reverted in https://github.com/LuxoftAKutsan/sdl_core/pull/113/commits/ab9adc7c6996e68e910941d6e3ed04f751e49215?w=1